src/org/usfirst/frc/team467/robot/Elevator.java

```diff
@@ -127,47 +125,59 @@ private void automaticMove(double heightInInches) {
 	 * @param speed The velocity. Shall be a value between -1 and 1.
 	 */
 	public void move(double speed) {
+		SmartDashboard.putNumber("Elevator/Move Speed", speed);
+
 		if (!RobotMap.HAS_ELEVATOR) {
 			return;
 		}
 
-		double currentHeight = getHeightInches();
 		// TODO Re-enable and test to see if it works
-		//		for (Stops stop : Stops.values()) {
-		//			if ((previousHeight < stop.height && currentHeight >= stop.height)
-		//					|| (previousHeight > stop.height && currentHeight <= stop.height)) {
-		//				DriverStation.getInstance().getNavRumbler().rumble(200, 0.8);
-		//			}
-		//		}
-
-		previousHeight = currentHeight;
+		// rumbleOnPresetHeights();
 
 		if (Math.abs(speed) >= RobotMap.MIN_LIFT_SPEED) {
 			// The controller is asking for elevator movement, cancel preset target and move.
 			targetHeight = null;
 			heightController.set(ControlMode.PercentOutput, speed);
 		} else if (targetHeight != null) {
 			// There is a target preset position, move there.
-			automaticMove(targetHeight.height);
+			automaticMove();
 		} else {
 			// Nothing to do, make sure we're not moving.
-			heightController.stopMotor();
+			heightController.disable();
+		}
+
+		limitCheck();
+		telemetry();
+	}
+
+	public void rumbleOnPresetHeights() {
+		double currentHeight = getRawHeight();
+		for(Stops stop: Stops.values()) {
+			if(stop.height == currentHeight) {
+				DriverStation.getInstance().getNavRumbler().rumble(150, 0.3);
+			}
 		}
 	}
 
-	public void logSensorAndTargetPosition() {
-		LOGGER.debug(
-				//TODO Check the arguments for the closed loop errors.
-				"Target= " + heightController.getActiveTrajectoryPosition()
-				+ "Pos=" + heightController.getSelectedSensorPosition(0));
+	public void limitCheck() {
```

Comment:
I'd like a Javadoc comment on the method describing what this does -- it's look for sensor slippage.